aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/requantization.h

```diff
@@ -128,48 +153,23 @@ static inline union pytorch_qnnp_conv_quantization_params
 pytorch_qnnp_compute_conv_quantization_params(
     uint8_t input_zero_point,
     uint8_t kernel_zero_point,
-    float scale,
+    float requantization_scale,
     uint8_t output_zero_point,
     uint8_t output_min,
     uint8_t output_max) {
-  /* Compute requantization parameters */
-  const uint32_t scale_bits = fp32_to_bits(scale);
-
-  /* Multiplier is in [0x40000000, 0x7FFFFF80] range */
-  const int32_t multiplier = (int32_t)(
-      ((scale_bits & UINT32_C(0x007FFFFF)) | UINT32_C(0x00800000)) << 7);
-  assert(multiplier >= INT32_C(0x40000000));
-  assert(multiplier <= INT32_C(0x7FFFFF80));
-
-  /* Shift is in [0, 31] range */
-  const int32_t shift = 127 + 31 - 32 - (fp32_to_bits(scale) >> 23);
-  assert(shift >= 0);
-  assert(shift < 32);
+  assert(scale > 0);
```

Comment:
Same questions about the scale range here too

Reply:
I think this constraint makes sense. I dont think scale is supposed to be < 0, right?

Reply:
Yeah this is ok, I meant the restriction that scale <=1. In the `pytorch_qnnp_compute_scalar_fp32_requantization_params` we have scale < 1.0f but here it is <=1. Why the difference?

Reply:
Oh good catch. This should be < 1.0 too although it will be removed so wont really matter.